mev-relay-rs/src/relay.rs

```diff
@@ -364,7 +367,7 @@ impl Relay {
             return Err(RelayError::InvalidFeeRecipient(
                 proposer_public_key.clone(),
                 fee_recipient.clone(),
-            ))
+            ));
```

Comment:
can we roll this back? I think it was leftover from an earlier cargo fmt not using nightly

Reply:
What do you mean by "this"?  

i updated my nightly build and run `just run-ci` but it doesn't suggest to add semicolons where you're wanting them to be added.

Reply:
Also, semicolons are already in place but don't show a difference from the previous commit.  

How did they appear without a trace?

Reply:
the change

so we don't need the semicolon here

I think you were running non-nightly cargo fmt and it added them

but if you ran this without doing that, it would leave them out

Reply:
Ahhh.  I see what you're saying.  

Not sure why, but when i run formatter with cargo nightly, it doesn't suggest removing the semicolons...

Reply:
its valid, but breaks convention with the repo